source/setup.py

```diff
@@ -110,80 +92,64 @@ def getRecursiveDataFiles(dest,source,excludes=()):
-	windows=[
-		{
-			"script":"nvda.pyw",
-			"dest_base":"nvda_noUIAccess",
-			"uiAccess": False,
-			"icon_resources":[(1,"images/nvda.ico")],
-			"other_resources": [], # Populated at run time
-			"version":formatBuildVersionString(),
-			"description":"NVDA application",
-			"product_name":name,
-			"product_version":version,
-			"copyright": NVDAcopyright,
-			"company_name":publisher,
-		},
-		# The nvda_uiAccess target will be added at runtime if required.
-		{
-			"script": "nvda_slave.pyw",
-			"uiAccess": False,
-			"icon_resources": [(1,"images/nvda.ico")],
-			"other_resources": [], # Populated at run time
-			"version":formatBuildVersionString(),
-			"description": name,
-			"product_name":name,
-			"product_version": version,
-			"copyright": NVDAcopyright,
-			"company_name": publisher,
-		},
-		{
-			"script": "nvda_eoaProxy.pyw",
-			# uiAccess will be enabled at runtime if appropriate.
-			"uiAccess": False,
-			"icon_resources": [(1,"images/nvda.ico")],
-			"other_resources": [], # Populated at run time
-			"version":formatBuildVersionString(),
-			"description": "NVDA Ease of Access proxy",
-			"product_name":name,
-			"product_version": version,
-			"copyright": NVDAcopyright,
-			"company_name": publisher,
-		},
-	],
+
+def _genManifestTemplate(shouldHaveUIAccess: bool) -> Tuple[int, int, bytes]:
+	return (
+		RT_MANIFEST,
+		1,
+		(_manifestTemplate % {"uiAccess": shouldHaveUIAccess}).encode("utf-8")
+	)
+
+
+_py2ExeWindows = [
+	{
+		"script": "nvda.pyw",
+		"dest_base": "nvda_noUIAccess",
+		"icon_resources": [(1, "images/nvda.ico")],
+		"other_resources": [_genManifestTemplate(shouldHaveUIAccess=False)],
+	},
+	# The nvda_uiAccess target will be added at runtime if required.
+	{
+		"script": "nvda_slave.pyw",
+		"icon_resources": [(1, "images/nvda.ico")],
+		"other_resources": [_genManifestTemplate(shouldHaveUIAccess=False)],
+	},
+	{
+		"script": "nvda_eoaProxy.pyw",
+		"icon_resources": [(1, "images/nvda.ico")],
+		"other_resources": [_genManifestTemplate(shouldHaveUIAccess=_partialArgs.uiAccess)],
+	},
+]
+if _partialArgs.uiAccess:
+	_py2ExeWindows.insert(1, {
+		"script": "nvda.pyw",
+		"dest_base": "nvda_uiAccess",
+		"icon_resources": [(1, "images/nvda.ico")],
+		"other_resources": [_genManifestTemplate(shouldHaveUIAccess=True)],
+	})
+
+
+freeze(
+	version_info={
+		"version": formatBuildVersionString(),
+		"description": description,
+		"product_name": name,
+		"product_version": version,
+		"copyright": NVDAcopyright,
+		"company_name": publisher,
+	},
+	windows=_py2ExeWindows,
 	console=[
 		{
 			"script": os.path.join("..", "include", "nvda_dmp", "nvda_dmp.py"),
 			"uiAccess": False,
 			"icon_resources": [(1, "images/nvda.ico")],
-			"other_resources": [],  # Populated at runtime
-			"version":formatBuildVersionString(),
-			"description": "NVDA Diff-match-patch proxy",
-			"product_name": name,
-			"product_version": version,
-			"copyright": f"{NVDAcopyright}, Bill Dengler",

```

Comment:
Previously nvda_dmp's binary had different copyright info than all other binaries. While this is obviously a  small think, and if it is difficult to support with the latest py2exe not  very important, just wanted to make sure this change was intentional.

Reply:
It turns out I was mistaken - we used to specify different version info for most of the binaries and not only for nvda_dmp. It seems this cannot be currently done with the new freeze interface to py2exe, as it accepts a single dictionary with version data in the freeze function. Would it be useful to report this to them, and perhaps invest in creating a fix?

Reply:
I'm assuming this is an intended change that the developers don't intend to fix, but it may be worth reporting

Reply:
Looking at the discussion in which the new freeze API was introduced I don't think this was done on purpose. I have created https://github.com/py2exe/py2exe/issues/193 to discuss if this can be reintroduced.

Reply:
It is now once again possible to specify different version info for each of the binaries in py2exe. Unfortunately while my fix is on py2exe's master, I have missed the release by a few hours. We can either note the fact that version info has changed as a known issue of this PR, and fix it up whenever next version of py2exe is released, or install it from git for the time being. Commit 4e7b2b2c60face592e67cb1bc935172a20fa371d, contains the fix but is otherwise identical to the latest official release.

Reply:
Thanks, this should be fixed now using that commit